__tests__/screens/send-confirmation.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+
+import { WalletCurrency } from "@app/graphql/generated"
+import { ConvertPaymentAmount } from "@app/screens/receive-bitcoin-screen/payment-requests/index.types"
+import SendBitcoinConfirmationScreen from "@app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen"
+import { act, render } from "@testing-library/react-native"
+import * as PaymentDetails from "../../app/screens/send-bitcoin-screen/payment-details/intraledger"
+import { Wrapping } from "./helper"
+
+const btcSendingWalletDescriptor = {
+  currency: WalletCurrency.Usd,
+  id: "testwallet",
+}
+
+const convertPaymentAmountMock: ConvertPaymentAmount = (amount, currency) => {
+  return {
+    amount: amount.amount,
+    currency,
+  }
+}
+
+const testAmount = {
+  amount: 100,
+  currency: WalletCurrency.Usd,
+}
+
+const defaultParams: PaymentDetails.CreateIntraledgerPaymentDetailsParams<WalletCurrency> =
+  {
+    handle: "test",
+    recipientWalletId: "testid",
+    convertPaymentAmount: convertPaymentAmountMock,
+    sendingWalletDescriptor: btcSendingWalletDescriptor,
+    unitOfAccountAmount: testAmount,
+  }
+
+const { createIntraledgerPaymentDetails } = PaymentDetails
+const paymentDetail = createIntraledgerPaymentDetails(defaultParams)
+
+const sendBitcoinConfirmation = {
+  key: "sendBitcoinConfirmationScreen",
+  name: "sendBitcoinConfirmation",
+  params: {
+    paymentDetail,
+  },
+} as const
+
+it("SendScreen Confirmation", async () => {
+  const { findByLabelText /* , toJSON */ } = render(
+    <Wrapping>
+      <SendBitcoinConfirmationScreen route={sendBitcoinConfirmation} />
+    </Wrapping>,
+  )
+  // it seems we need multiple act because the component re-render multiple times
+  // probably this could be debug with why-did-you-render
+  await act(async () => {})
+  await act(async () => {})
+
+  // console.log(JSON.stringify(toJSON(), null, 2))
+  const { children } = await findByLabelText("Successful Fee")
+  expect(children).toEqual(["₦0.00 - $0.00"])
```

Comment:
correct. I'm showing how this can be used eventually, this is not meant to be a "real test" that will catch bug. but eventually we should build on it.